packages/frint-router-react/src/Route.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getMountableComponent } from 'frint-react';
+/* eslint-enable import/no-extraneous-dependencies */
+
+export default class Route extends React.Component {
+  static contextTypes = {
+    app: PropTypes.object.isRequired
+  };
+
+  static propTypes = {
+    path: PropTypes.string,
+    exact: PropTypes.bool,
+    computedMatch: PropTypes.object,
+    component: PropTypes.func,
```

Comment:
@jackTheRipper removed the `PropTypes.node`, because `PropTypes.func` is sufficient